ui/ducks/send/send.js

```diff
@@ -1383,6 +1384,7 @@ const slice = createSlice({
             (!isValidHexAddress(state.recipientInput, {
               mixedCaseUseChecksum: true,
             }) &&
+              !IS_FLASK &&
```

Comment:
Why is this needed?

Reply:
`onChange` prop func in `DomainInput` calls `updateRecipientUserInput` which makes its way to this line of code and erroneously assigns an error if not hex in flask before we even get to the domain lookup call.